config/locales/en.yml

```diff
@@ -2,23 +2,31 @@
 en:
   global_popup_title: "Redmine Advanced Messenger"
   
-  issue_notifications_dialog_go_to_first_one: "Go to first one"
-  issue_notifications_dialog_text_for_current_user: "For %{user}, there are <span id='unread-notifications-count'></span> <span style='color: red'> unread </span> notifications."
-  issue_notifications_dialog_others_tooltip_text: "Unread notifications"
-  issue_notifications_dialog_others_tooltip_text_all_read: "Other users have all messages read."
-  issue_notifications_dialog_status_for_others: "Status for other users"
-  
-  journal_collapse_note_message: "The message is collapsed. Preview: "
-  journal_collapse_note_message_read: "The message is read, but collapsed. Preview: "
-  journal_collapse_note_message_unread: "The message is unread. Please expand it to see its content & mark read."
-  journal_notifications_tooltip_list_title: "%{date} notifications sent (<span style='color:green'>read</span>/<span style='color:red'>unread</span>) to:"
-  journal_notifications_tooltip_list_item_not_viewed: "unread"
-  journal_read: "Read"
-
   label_time_ago: "%{time} ago"
+  
+  message_collapsed: "The message is collapsed. Preview: "
+  message_collapsed_ignored: "The message is ignored."
+  message_collapsed_read: "The message is read, but collapsed. Preview: "
+  message_collapsed_unread: "The message is unread. Please expand it to see its content & mark read."
+  message_notifications_tooltip_list_title: "%{date} notifications sent (<span style='color:green'>read</span>/<span style='color:red'>unread</span>/<span style='color:#ccc'>ignored</span>) to:"
+  message_notifications_tooltip_list_item_not_viewed: "unread"
+  message_ignored: "Ignored"
+  message_read: "Read"
+  message_read_but_collapsed: "Read but collapsed"
+  message_unread: "Unread"
+  
+  notifications_dialog_go_to_first_one: "Go to first one"
+  notifications_dialog_text_for_current_user: "For %{user}, there are <span id='unread-notifications-count'></span> <span style='color: red'> unread </span> notifications."
+  notifications_dialog_others_tooltip_text: "Unread notifications"
+  notifications_dialog_others_tooltip_text_all_read: "Other users have all messages read."
+  notifications_dialog_status_for_others: "Status for other users"
 
-  my_page_unread_notifications: "Unread notifications"
-  my_page_all_notifications_read: "All notifications are read"
+  my_page_issues_unread_notifications: "Unread issues notifications:"
+  my_page_forum_unread_notifications: "Unread forum messages:"
+  my_page_all_issues_notifications_read: "All issues notifications are read"
+  my_page_all_forum_notifications_read: "All forum messages are read"
+  my_page_mark_all_as_ignored: "Mark all as ignored"
```

Comment:
I think it should have been `"Mark all unread as ignored"`, right?

Reply:
I have modified this